redis/RedisCluster.php

```diff
@@ -2506,7 +2506,7 @@ public function zUnionStore($Output, $ZSetKeys, array $Weights = null, $aggregat
      * $redisCluster->zInterStore('ko4', array('k1', 'k2'), array(1, 5), 'max'); // 2, 'ko4' => array('val3', 'val1')
      * </pre>
      */
-    public function zInterStore($Output, $ZSetKeys, array $Weights = null, $aggregateFunction = 'SUM') { }
+    public function zInterStore($Output, $ZSetKeys, ?array $Weights = null, $aggregateFunction = 'SUM') { }
```

Comment:
As far as parent class isn't final I suggest leaving parameter type hint non-nullable to avoid extra warning with PHP < 7.1 in child classes